envoyauth/evaluation.go

```diff
@@ -21,6 +21,7 @@ type EvalContext interface {
 	Store() storage.Store
 	Compiler() *ast.Compiler
 	Runtime() *ast.Term
+	Target() string
```

Comment:
This would be a breaking change..correct ?

Reply:
Indeed! I'll see what I can do about it.

Reply:
Gave this another go. What do you think?